docs/docs/tools-overview.md

```diff
@@ -185,7 +185,7 @@ This powerful feature enables you to create hierarchical agent architectures whe
 
 ### Converting agents to tools
 
-To convert an agent into a tool, use the `asTool()` extension function:
+To convert an agent into a tool, use the `createAgentTool()` extension function:
```

Comment:
Thanks for catching this. However, this is extension on the `AIAgentService`, not `AIAgent` itself. So the corrected version should look something like that:
> [...] use `AIAgentService` and `createAgentTool` extension function